config.yaml

```diff
@@ -13,19 +13,21 @@ dataset:
   max_queries: 1000
   min_input_tokens: 0
   max_input_tokens: 1024
-  max_output_tokens: 256
-  max_sequence_tokens: 1024
+  min_output_tokens: 0
+  max_output_tokens: 1024
+  max_sequence_tokens: 2048
 load_options:
   type: constant #Future options: loadgen, stair-step
-  concurrency: 2
+  concurrency: 1
   duration: 20 # In seconds. Maybe in future support "100s" "10m", etc...
-plugin: "tgis_grpc_plugin"
+plugin: "openai_plugin"
 plugin_options:
-  #interface: "grpc" # Some plugins like caikit-nlp-client should support grpc/http
+  # interface: "grpc" # Some plugins like caikit-nlp-client should support grpc/http
   use_tls: False # Use True if querying an SSL grpc endpoint over https
   streaming: True
   model_name: "flan-t5-small"
   host: "route.to.host"
-  port: 8033
+  endpoint: "/v1/completions" # for openai plugin
+  # port: 8033 #
```

Comment:
Maybe remove the `port` and `interface` comments since the openai_plugin does not support them.

Reply:
I added some context to hopefully make it clear that only some plugins require them. WDYT?